edgelet/edgelet-config/src/lib.rs

```diff
@@ -440,6 +529,18 @@ mod tests {
     static GOOD_SETTINGS_TG: &str = "test/windows/sample_settings.tg.yaml";
     #[cfg(windows)]
     static GOOD_SETTINGS_DPS_SYM_KEY: &str = "test/windows/sample_settings.dps.sym.yaml";
+    #[cfg(windows)]
+    static GOOD_SETTINGS_DPS_TPM: &str = "test/windows/sample_settings.dps.tpm.yaml";
```

Comment:
Minor: Can you add a comment to describe what this file is actually testing vs the other TPM files? Applicable to other files as well. It can assist maintainers to understand what is going on on a high level.

Reply:
As discussed, I'll add to this to master